client.lua

```diff
@@ -9,37 +10,33 @@ end)
 
 RegisterNetEvent('esx:armor')
 AddEventHandler('esx:armor', function()
-	local playerPed  = GetPlayerPed(-1)
-	local armor = loadModel(GetHashKey('prop_bodyarmour_03'))
-	local crate = CreateObject(armor, GetEntityCoords(PlayerPedId()), true, false, false)
+	while not HasModelLoaded(armorModel) do

```

Comment:
need give this function hash of model so you should use ` or GetHashKey
but still functions make code cleaner